dind/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: '3'
+services:
+
+    cluster1:
```

Comment:
As mentioned earlier cluster doesn't appear a proper term do you mind to use ``node1``, ``node2``... ?
 
> **note**: cluster_lab repo and integration test use ``core1``, ``core2``... but as dind docker container are not based on coreos I would prefer ``node[1-4]``, you'll have to adapt that I guess